src/components/Icon.tsx

```diff
@@ -70,5 +71,5 @@ export default function Icon({ icon, size = 20, className }: IconProps) {
         <path key={index} fillRule="evenodd" d={path} />
       ))}
     </svg>
-  ) : null;
```

Comment:
Safe to assume this won't ever be null, since the ts compiler references the keys of the icons object